s3auth-relay/pom.xml

```diff
@@ -174,6 +174,11 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-net</groupId>
+            <artifactId>commons-net</artifactId>
+            <version>3.3</version>
```

Comment:
so what about this? I think @krzyk is right, the version is not required here, it is inherited from parent POM: https://github.com/jcabi/jcabi-parent/blob/master/pom.xml#L1221-L1229